### PR TITLE
Spelling and Grammar Correction.

### PR DESCRIPTION
**Pull request by KawaneRio**
Did it more thorough this time... I think.

### CODE REVIEW

File: src/reference/controlflow.md [hunk 1]
```diff
@@ -104,7 +104,7 @@ The compiler then transforms the statement into byte code equivalent to the
 statement rewritten to
 
 ```
-🍡 iterable❗️ ➡️ iterator
+🍡 iteratable❗️ ➡️ iterator
```
Comment (thbwd) [inline src/reference/controlflow.md]: The word "iterable" is commonly used. See for example [Java](https://docs.oracle.com/javase/8/docs/api/java/lang/Iterable.html) or [JavaScript](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Iteration_protocols). I don't think I've read "iteratable" before.

**Review by thbwd — COMMENTED** [review]
Thanks for these improvements. Can you please revert the "iterable" thing?

**Review by KawaneRio — COMMENTED** [review]
┏━━━━━━━━━━━┳━━┓
┃File revert GitHub HOW  ┃🔎　┃
┗━━━━━━━━━━━┻━━┛

**Comment by thbwd** [comment]
I don't think reverting commits is possible on GitHub 🤷‍♀️ Or is it?

**Pull request merged**